fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -380,7 +380,8 @@ class GripperTeleop : public TeleopComponent
     ros::NodeHandle pnh(nh, name);
 
     // Button mapping
-    pnh.param("button_deadman", deadman_, 10);
+    pnh.param("button_deadman", deadman_body_, 9);
+    pnh.param("button_deadman", deadman_end_, 11);
```

Comment:
same parameter name as previous parameter, should have a different name (perhaps "button_deadman_arm")